### DISCUSSION TITLE
initial start-up and shutdown

### DISCUSSION THREAD

**Discussion by AlDo2358** [lllyasviel/Fooocus#2541] [Q&A]
The installation was successful. Fooocus stopped after the model download was complete 
6.62G/6.62G [42:11<00:00, 2.93MB/s]. 
It's been >2 hours. Leaving it overnight. Please advise on how to fix this. Space on the installation disc is 300GBt. Xeon 4620, 16GBt, AMD RX5700 XT 8GBt. 
![pic1](https://github.com/lllyasviel/Fooocus/assets/158803611/ffb77726-7543-4cee-b0bf-fb70d3f232df)

**Comment by mashb1t** [comment]
Please check if restarting Fooocus works for you (ehich it should).

**Reply by AlDo2358** [reply]
I think it was my internet problem. All the models installed today. One model was downloaded manually, because FS could not finish the download itself. Now there is a message (see attachment). Can you please tell me how to fix it?
![pic1](https://github.com/lllyasviel/Fooocus/assets/158803611/faeffda8-5481-485b-b900-9360ec167d14)

**Reply by mashb1t** [reply]
Please follow the installation instructions for AMD GPUs in the readme.

**Reply by AlDo2358** [reply]
Thank you, it worked. Fooocus started without errors. I'm glad. First image in Opera - crashed. Second FireFox - successfully. The system window of Fooocus after loading maximises memory and disk load. Sending a screenshot. Computer stopped. Thanks for your help anyway.
![pic2](https://github.com/lllyasviel/Fooocus/assets/158803611/ef47734e-256f-472e-b237-21f355c2e82b)
